components/ui-api-layer/pkg/gqlerror/errors.go

```diff
@@ -183,3 +198,9 @@ func appendArgument(arguments, name, value string) string {
 
 	return result
 }
+
+func buildInvalidErrorMsg(msg string) string {
```

Comment:
it is not a good idea to change the message., let's have an original one even if it is not so pretty

Reply:
Ok, I removed that. Now I return a raw error.